palisade-service/README.md

```diff
@@ -18,39 +18,50 @@ limitations under the License.
 
 # Palisade Service
 
-The Palisade service is the entry point into the rest of the Palisade services, it accepts a request from the client and passes it forwards to the User Service to continue on the processing of the request. The client then receives a token used to connect
-to the Filtered Resource Service and retrieve the processed request.
+The Palisade Service is the entry point into the rest of the Palisade Services, it accepts a request from the client, containing the userId of the user requesting access, 
+the resource the user wants access to, and a reason as to why they require access. 
+The Palisade Service then creates a token, currently via the UUIDPalisadeService, but this can be extended in the future for specific token requirements. 
+The token is attached to the message in the form of a `PalisadeSystemResponse` to be sent to the User Service on the `user` kafka topic. 
+It is also sent back to the client directly as part of a `PalisadeClientResponse` so that the client can connect to the Filtered Resource Service to retrieve the processed data.
+As the data progresses through the other Palisade services, it is refined and enriched, the Palisade Service is just the entry point for client requests.
+
+## High-Level Architecture
+<!--- 
+See palisade-service/doc/palisade-service.drawio for the source of this diagram
+--->
+![Palisade Service diagram](doc/palisade-service.png)
+The routing of requests is shown in the diagram above. The yellow boxes indicate the client, and the purple are kafka topics.
```

Comment:
Header PalasadeRequest in the table needs to be changed to PalisadeClientRequest